VocaDbModel/Database/Queries/UserQueries.cs

```diff
@@ -191,13 +192,8 @@ private CachedUserStats GetCachedUserStats(IDatabaseContext<User> ctx, User user
 					stats.ArtistCount = GetArtistCount(ctx, user);
 					stats.FavoriteSongCount = GetSongCount(ctx, user);
 
-					stats.CommentCount
-						= ctx.Query<AlbumComment>().Count(c => c.Author.Id == user.Id)
-						+ ctx.Query<ArtistComment>().Count(c => c.Author.Id == user.Id)
-						+ ctx.Query<SongComment>().Count(c => c.Author.Id == user.Id);

```

Comment:
I suspect the original implementation was wrong.

Reply:
If you're referring it to not counting all entry types, I probably just didn't update it when adding more comment types. It's only informational anyway.